redgenes/gene_annotations.py

```diff
@@ -183,11 +192,11 @@ def extract_barrnap_results(gff_path):
 def annotation_pipeline(df, tmpdir, kofamscan_profile, kofamscan_kolist):
     for _, row in df.iterrows():
         # Prepare data for 'identifier' table and 'md_info' table
-        local_path = row["local_path"]
-        filename = row["assembly_accession"]
+        local_path = row["local_path"].strip()
+        filename = row["assembly_accession"].strip()
 
-        source = row["source"]
-        external_accession = row["assembly_accession"]
+        source = row["source"].strip()
```

Comment:
It looks like this is iterating over a dataframe. If loaded with `read_csv`, there shouldn't be any newlines?

Reply:
I was just using `strip()` in case there were leading/trailing whitespaces in the input. This is probably not necessary(?)